src/sensors/odometer/interrupt/DirectionlessOdometer.cpp

```diff
@@ -38,7 +38,7 @@ float DirectionlessOdometer::getSpeed()
 {
     // To get the current speed in m/sec, divide the meters per pulse (dx) with
     // the length between the last two pulses (dt)
-    return mDt > 0 ? kMillisecondsInSecond * mMillimetersPerPulse / static_cast<float>(mDt) : 0.0F;
+    return mDt > 0 ? kMillisecondsInSecond * mMillimetersPerPulse / static_cast<float>(mDt) : 0.0f;
```

Comment:
:warning: **readability-uppercase-literal-suffix** :warning:
floating point literal has suffix 'f', which is not uppercase
```suggestion
    return mDt > 0 ? kMillisecondsInSecond * mMillimetersPerPulse / static_cast<float>(mDt) : F```